### PR TITLE
Adds Postgres sql script to add timestamps to 'sim_entries' table

### PR DESCRIPTION
**Pull request by kmmatwork**
Work in progress for the time being.

### CODE REVIEW

File: sim-administration/simmanager/schema/add-timestamps.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+returns trigger as $$
+begin
+  new.tsprovisionstate := now();
+  return new;
+end;
+$$ language plpgsql;
+create trigger trigger_provisionstate
+before update on sim_entries
+for each row
+when (old.provisionstate is distinct from new.provisionstate)
```
Comment (la3lma) [inline sim-administration/simmanager/schema/add-timestamps.sql]: Would a bit of indentation be possible here? (I don't know sql so well), but illustrating the block structure of the for loop could be useful (that would be true for the other for loops too).

File: sim-administration/simmanager/schema/add-timestamps.sql [hunk 2]
```diff
@@ -0,0 +1,53 @@
+--
+-- Add timstamp column for hlrstate
```
Comment (la3lma) [inline sim-administration/simmanager/schema/add-timestamps.sql]: I really like this approach!  Very elegant.

**Review by la3lma — APPROVED** [review]

File: sim-administration/postgres/init.sql [hunk 3]
```diff
@@ -22,6 +29,9 @@ create table sim_entries (id bigserial primary key,
                           pin2 varchar(4),
                           puk1 varchar(8),
                           puk2 varchar(8),
+                          tsHlrState timestamp default now(),
+                          tsSmdpPlusState timestamp default now(),
+                          tsProvisionState timestamp default now(),
```
Comment (vihangpatil) [inline sim-administration/postgres/init.sql:34]: So, these are not the timestamps for every state transition, but instead the timestamp when the last state transition happened for that state.  If that's sufficient for the use-case,  then good! 👍

**Review by vihangpatil — APPROVED** [review]

File: sim-administration/postgres/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# SIM Manager DB schema
```
Comment (la3lma) [inline sim-administration/postgres/README.md:1]: Please take a look at  the Codacy suggestions for the markdown file.  They are minor, but will probably make the markdown a little stricter.

**Pull request merged**